azavea/pfb-network-connectivity
# Add crash markers with popups to UI

## Overview

This PR adds crash data to the client

### Notes

I had to make a couple of compromises
1. I moved the legends to the bottom left so they wouldn't interfere with the layer picker
2. I didn't add a new legend and instead added a hint to the other active transportation markers

### Demo

![Screenshot from 2022-12-07 12-01-24](https://user-images.githubusercontent.com/36001800/206270004-1b9c43a5-2c75-42af-bbab-8114a36976ec.png)



## Testing Instructions

 * Run `scripts/update` so you can get the new leaflet dependency
 * Start up the server
 * Go to a place where you have an analysis job, e.g. `http://localhost:9301/#/places/some_analysis_job_uuid`
 * Open the layer picker and select crashes
 * You should be able to see crash markers and clustered markers on the map
 * You should be able to see popups when you click on the markers that show year, fatalities, and an optional hint if the crash is an active transportation crash of any kind, and another optional hint if it is either an active transportation crash or a bike crash with multiple fatalities


## Checklist


Resolves #921
Resolves #922 
Resolves #923
